mautic/mautic
# Api rate limiter

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? | N
| Related user documentation PR URL | 
| Related developer documentation PR URL | https://github.com/mautic/developer-documentation/pull/113
| Issues addressed (#s or URLs) | 
| BC breaks? | N
| Deprecations? | N

[//]: # ( Required: )
#### Description:
Add an input in API configuration tab to specify hourly api rate limit

#### Steps to test this PR:
1. Set 2 to api rate limit
2. Make 3 api call, and see message and 429 HTML code
3. wait 1hour, and make a new API call.
